cve_bin_tool/cli.py

```diff
@@ -283,6 +284,10 @@ def scan_files_unpack(unpacked):
 
 def main(argv=None):
     """ Scan a binary file for certain open source libraries that may have CVEs """
+
+    # update egg before performing any operation
+    update_egg()
```

Comment:
If this is the only place update_egg() is ever going to be called, I don't think it needs a separate file (and it may not even need a separate function, but I don't mind that so much).  We actually need the egg updated just befor the checkers are loaded, so it probably could be grouped with the checker initialization functions.

Reply:
Yeah I have tried to put it in cli.py but it isn't working because it look for relative path from where we call it. To address this, we have to keep it at the root level.